.github/workflows/publish-snapshot.yml

```diff
@@ -41,7 +41,7 @@ jobs:
                       ${{ runner.os }}-
 
             - name: Install Dependencies
-              run: npm install
+              run: make setup
 
             - name: Build Pyscript.core
               run: npm run build
```

Comment:
I think here you need `npx playwright install` before `npm run build` here

Reply:
I've put it as part of the "install deps" step (feels like the right place to be).

Reply:
it's OK, that should work too